Mathlib/NumberTheory/NumberField/Ideal/KummerDedekind.lean

```diff
@@ -0,0 +1,119 @@
+/-
+Copyright (c) 2025 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.NumberTheory.KummerDedekind
+import Mathlib.NumberTheory.NumberField.Basic
+import Mathlib.RingTheory.Ideal.Norm.AbsNorm
+import Mathlib.RingTheory.Ideal.Int
+
+/-!
+# Kummer-Dedekind criterion for the splitting of prime numbers
+
+In this file, we give a specialized version of the Kummer-Dedekind criterion for the case of the
+splitting of rational primes in number fields.
+
+## Main definitions
+
+* `RingOfIntegers.exponent`: the smallest positive integer `d` contained in the conductor of `θ`.
+  It is the smallest integer such that `d • 𝓞 K ⊆ ℤ[θ]`, see `RingOfIntegers.exponent_eq_sInf`.
+
+* `RingOfIntegers.ZModXQuotSpanEquivQuotSpan`: The isomorphism between `(ℤ / pℤ)[X] / (minpoly θ)`
+  and `𝓞 K / p(𝓞 K)` for a prime `p` which doesn't divide the exponent of `θ`.
+
+## Main results
+
+* TODO
```

Comment:
You can maybe add the statement (keeping the TODO of course).

Reply:
I removed the section. I'll add it back in the next PR where the main results are proved